packages/cfd/src/features/hooks/useCurrentList.ts

```diff
@@ -0,0 +1,51 @@
+import { useStore } from '@deriv/stores';
+import { useMT5AccountsList, useDxtradeAccountsList, useCtraderAccountsList } from '@deriv/api';
+import { getAccountListKey } from '@deriv/shared';
+
+type MT5AccountsList = NonNullable<ReturnType<typeof useMT5AccountsList>['data']>[number];
+type DxtradeAccountsList = NonNullable<ReturnType<typeof useDxtradeAccountsList>['data']>[number];
+type CtraderAccountsList = NonNullable<ReturnType<typeof useCtraderAccountsList>['data']>[number];
+
+type TCurrentList = {
+    [key: string]: MT5AccountsList | DxtradeAccountsList | CtraderAccountsList;
+};
+
+/* 
+    This hook is used to get the existing_accounts_data
+*/
+const useCurrentList = () => {
+    const current_list: TCurrentList = {};
+    const { traders_hub } = useStore();
+    const { data: mt5_accounts } = useMT5AccountsList();
```

Comment:
We can use the CFDAccountsList because here we are using every platform so combined one make more sense.
If we want we can destructure from the `cfd_accounts`.
Just a suggestion.
```suggestion
    const { data: cfd_accounts } = useCFDAccountsList();
```